# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

RobbieArmActionGoal action_goal
RobbieArmActionResult action_result
RobbieArmActionFeedback action_feedback
